kafka-connect-cloud-common/src/main/scala/io/lenses/streamreactor/connect/cloud/common/sink/naming/CloudKeyNamer.scala

```diff
@@ -199,10 +199,7 @@ class CloudKeyNamer(
 
   private def getFieldStringValue(struct: SinkData, partitionName: Option[PartitionNamePath]) =
     adaptErrorResponse(SinkDataExtractor.extractPathFromSinkData(struct)(partitionName)).fold(Option.empty[String])(
-      fieldVal =>
-        Option(fieldVal
-          .replace("/", "-")
-          .replace("\\", "-")),
+      fieldVal => Option(fieldVal),
```

Comment:
Actually you can simplify this further!

adaptErrorResponse(SinkDataExtractor.extractPathFromSinkData(struct)(partitionName))

You don't need the fold or anything after